src/goto-programs/goto_inline_class.cpp

```diff
@@ -853,28 +853,32 @@ bool goto_inlinet::check_inline_map(
   if(call_list.empty())
     return true;
 
+  #if 0
   int ln=-1;
+  #endif
 
   for(call_listt::const_iterator c_it=call_list.begin();
       c_it!=call_list.end(); c_it++)
   {
     const callt &call=*c_it;
     const goto_programt::const_targett target=call.first;
 
-#if 0
+    #if 0
     // might not hold if call was previously inlined
     if(target->function!=identifier)
       return false;
-#endif
 
     // location numbers increasing
     if(static_cast<int>(target->location_number)<=ln)
       return false;
+    #endif
 
     if(!is_call(target))
       return false;
 
+    #if 0
```

Comment:
This can definitely be deleted since it appears to be a modification of a local variable just before it goes out of scope.